src/bundle/Resources/config/services.yml

```diff
@@ -26,6 +26,13 @@ services:
             - [setContainer, ["@service_container"]]
             - [performAccessCheck, []]
 
+    EzSystems\EzPlatformAdminUiBundle\Controller\LocationController:
+        tags: [controller.service_arguments]
+        calls:
+            - [setContainer, ['@service_container']]
+        arguments:
+            $copySubtreeLimit: '%ezsettings.default.application_config.copy_subtree.limit%'
```

Comment:
please inject it using dynamic settings. In this case it would be `$application_config.copy_subtree.limit$` but please see the previous comment.